chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -968,6 +1112,8 @@ def get_settings(self):
         dtmfe = RadioSettingGroup("dtmfe", "DTMF Encode Settings")
         self.get_settings_common_dtmf(dtmfe, _mem)
         self.get_settings_pro_dtmf(dtmfe, _mem)
+        if self.MODEL == "BF-F8HP_Pro":
+            self.get_settings_f8hp_pro_dtmf(dtmfe, _mem)
```

Comment:
It would be much better to just override `get_settings_f8hp_pro_dtmf()` in the F8HP subclass and add the extra stuff there. I know this is the pattern you're used to, but in this case, this stuff is starting to get a bit messy as a result, so it'd be better to move that there. Using `super()` will let you call the parent implementation and then add on more so it's all localized down there.